src/pages/Home.vue

```diff
@@ -6,15 +6,56 @@
     <form>
       <p>Select path to LevelDB</p>
       <label class="file db-path">
-        <input type="file" id="file" aria-label="File browser example" />
-        <span class="file-custom"></span>
+        <input type="text" id="file" webkitdirectory />
```

Comment:
@emanlodovice `<input>` element is not necessary anymore if we display the selected path in the `<span>` below. Can we remove it and its corresponding styles?